oaknational/Oak-Web-Application
# feat: adds school picker

## Description

- adds useSchoolPicker hook to fetch schools list
- adds SearchComboBox component
- adds SchoolPicker
- Adds school picker to downloads page
- Adds logic for radio buttons and school picker
-


## Issue(s)

Fixes #1110
Fixes #1221

## How to test

1. Go to downloads page
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
